### PR TITLE
Default task + param names to be humanized slug

### PR DESCRIPTION
**Pull request by ericliuche**
Before, the default task and parameter names were just the task and parameter slugs. Now, we first humanize the slug for a prettier name

### CODE REVIEW

File: packages/airplane/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "@azure/abort-controller": "1.1.0",
     "@opentelemetry/api": "1.2.0",
     "fast-equals": "4.0.3",
+    "humanize-string": "^3.0.0",
```
Comment (colinking) [inline packages/airplane/package.json:41]: pin this dep

File: packages/airplane/internal/config.ts [hunk 2]
```diff
@@ -357,7 +358,7 @@ export const workflow = <TParams extends InputParams, TOutput>(
 
   wrappedF.__airplane = {
     type: "workflow",
-    config: dedentConfig(config),
+    config: normalizeConfig(config),
```
Comment (colinking) [inline packages/airplane/internal/config.ts:361]: Add this to Views, too (just for the view name, since there are no parameters yet).

**Review by colinking — APPROVED** [review]
Nice!

File: packages/airplane/internal/config.ts (hunk 2 above)
Comment (colinking) [reply packages/airplane/internal/config.ts:361]: Yeah, IMO it should, to match tasks. I doubt that was intentional.

Comment (ericliuche) [reply packages/airplane/internal/config.ts:361]: 👍

**Comment by leeweisberger** [comment]
I was just gonna do part this and suddenly it works!

**Pull request merged**